compiler/noirc_frontend/src/monomorphization/debug.rs

```diff
@@ -39,10 +39,10 @@ impl Monomorphizer<'_> {
     /// different types for each instance at runtime.
     pub(super) fn patch_debug_instrumentation_call(
         &mut self,
+        original_func: Box<Expression>,
```

Comment:
Nitpick: we don't need to move `original_func` into this function either. Passing a reference should eliminate the clone when calling this function. See https://github.com/noir-lang/noir/pull/9484/files#diff-cf8aebeffeb8bd0425789bfa0bfcb832252a60340de52377b8bbc1a2ee240c1dR43

Reply:
Done! Now it's written the same way as in your PR.